Source/AwsCommonRuntimeKit/sdkutils/CRTAWSProfileCollection.swift

```diff
@@ -24,34 +26,39 @@ public class CRTAWSProfileCollection {
         self.rawValue = profilePointer
     }
 
-    public init(fromBuffer buffer: ByteBuffer,
-                source: CRTAWSProfileSourceType,
-                allocator: Allocator = defaultAllocator) {
+    /// Create a new profile collection by parsing text in a buffer. Primarily for testing.
+    init(fromBuffer buffer: ByteBuffer,
```

Comment:
this `init()` should be `throws` or `?` like the other init functions. `aws_profile_collection_new_from_buffer()` could fail

Reply:
Thanks, fixed.